File: Model/Booking.java
package Model;

public class Booking {
    private int bookingNumber;

    public Booking(int bookingNumber){
        this.bookingNumber = bookingNumber;
    }

    public int getBookingNumber() {
        return bookingNumber;
    }

    public void setBookingNumber(int bookingNumber) {
        this.bookingNumber = bookingNumber;
    }


    public boolean equals(Booking anotherBooking) {
        return this.bookingNumber == anotherBooking.bookingNumber;
    }
}

File: Model/Booking.py
class Booking:
    def __init__(self, booking_number):
        self.booking_number = booking_number

    def get_booking_number(self):
        return self.booking_number

    def set_booking_number(self, booking_number):
        self.booking_number = booking_number

    def __eq__(self, other):
        if isinstance(other, Booking):
            return self.booking_number == other.booking_number
        return False

